src/main/kotlin/org/oso/core/services/impl/DefaultMailService.kt

```diff
@@ -0,0 +1,40 @@
+package org.oso.core.services.impl
+
+import org.oso.core.services.MailService
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.mail.SimpleMailMessage
+import org.springframework.mail.javamail.JavaMailSender
+import org.springframework.mail.javamail.JavaMailSenderImpl
+import org.springframework.stereotype.Service
+
+@Service
+class DefaultMailService
+    /**
+    * @constructor
+    * @param mailSender Spring creates this bean automatically when it is not defined.
+     *                  It takes the mail properties as program argument or from the application.yml.
+    */
+    @Autowired constructor(private val mailSender: JavaMailSender) : MailService {
+    private val FROM = "kontakt@ososystem.de"
```

Comment:
i think this could be in the yml instead

Reply:
I moved it :-)